app/views/providers/show.html.erb

```diff
@@ -7,48 +7,44 @@
-                          provider_recruitment_cycle_path(@provider.provider_code, '2019'),
-                          data: { qa: 'provider__courses__current_cycle' },
-                          class: "govuk-link" %>
-      </h2>
-      <p class="govuk-body">Use this section to:</p>
-      <ul class="govuk-list govuk-list--bullet govuk-!-margin-bottom-8">
-        <li>manage courses in the current cycle</li>
-        <li>edit locations and vacancies in the current cycle</li>
-      </ul>
-
-      <h2 class="govuk-heading-m">
-        <%= link_to 'Next cycle (2020 - 2021)',
-                          provider_recruitment_cycle_path(@provider.provider_code, '2020'),
-                          data: { qa: 'provider__courses__next_cycle' },
-                          class: "govuk-link" %>
-      </h2>
-      <p class="govuk-body">Use this section to:</p>
-      <ul class="govuk-list govuk-list--bullet govuk-!-margin-bottom-8">
-        <li>prepare courses for the next cycle</li>
-        <li>manage locations needed for the next cycle</li>
-      </ul>
-    <% end %>
-
-    <% unless @provider.rolled_over? %>
-      <%= render partial: 'recruitment_cycles/courses_info', locals: { provider: @provider, year: '2019' } %>
-      <%= render partial: 'recruitment_cycles/locations_info', locals: { provider: @provider, year: '2019' } %>
-    <% end %>
-
-    <h2 class="govuk-heading-m"><%= govuk_link_to "Request access for someone else", request_access_provider_path(code: @provider.provider_code) %></h2>
-    <p class="govuk-body govuk-!-margin-bottom-8">You can request a DfE Sign-in account for others who manage your courses.</p>
-
-    <h2 class="govuk-heading-m"><%= govuk_link_to "UCAS contacts", ucas_contacts_provider_path(code: @provider.provider_code) %></h2>
-    <p class="govuk-body govuk-!-margin-bottom-8">You can update your contacts for UCAS.</p>
+    <h2 class="govuk-heading-l">New cycle (2020 - 2021)</h2>
+    <p class="govuk-body">Before 1 October you must:</p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>publish your organisations information</li>
+      <li>publish the courses you are running this year</li>
+      <li>remove any mention of the skills test requirement</li>
+    </ul>
+    <p class="govuk-body">Courses you publish will be listed on <%= govuk_link_to "Find postgraduate teacher training", "" %> from 1 October. Applications will open on 8 October.</p>
+    <p class="govuk-body govuk-!-margin-bottom-8">Candidates can only apply to published courses.</p>
+
+    <%= render partial: 'recruitment_cycles/about_organisation', locals: { provider: @provider, year: '2020' } %>
+    <%= render partial: 'recruitment_cycles/courses_info', locals: { provider: @provider, year: '2020' } %>
+    <%= render partial: 'recruitment_cycles/locations_info', locals: { provider: @provider, year: '2020' } %>
+
+    <h2 class="govuk-heading-l">Previous cycles</h2>
+
+    <h3 class="govuk-heading-m">
+      <%= link_to '2019 - 2020',
```

Comment:
This link doesn’t work. It keeps redirecting back to the organisation page.

Reply:
Fixed this bug